lib/core/config.rb

```diff
@@ -96,14 +100,7 @@ def apps
   def current
     return unless app
 
-    @current ||= begin
-      app_config = find_app_config(app)
-      ensure_config_app!(app, app_config)
```

Comment:
We don't need this because we already do it in `apps`, which `find_app_config` calls.